components/automate-chef-io/content/docs/iam-v2-overview.md

```diff
@@ -1,5 +1,10 @@
 +++
 title = "IAM Overview"
+aliases = [
+    "/docs/authorization-overview/",
+    "/docs/default-policies/",
+    "/docs/iam-v1-policies/"
```

Comment:
@susanev Does another alias need to be added for "/docs/iam-v1-overview/" too?

Reply:
yes, good call thank you!